configure.ac

```diff
@@ -755,6 +755,10 @@ AC_CHECK_LIB(boost_system-mt, main, [],
     [AC_CHECK_LIB(boost_system, main, [],
         AC_MSG_NOTICE(["Boost system library not found."]))])
 
+AC_CHECK_LIB(boost_iostreams-mt, main, [],
+    [AC_CHECK_LIB(boost_iostreams, main, [],
+        AC_MSG_NOTICE(["Boost iostreams library not found."]))])
```

Comment:
Is this a new package?  Ideally iostreams-mt is already part of the boost libraries in build-depends in debian/control and ceph.spec.in ....